sensor/kubernetes/listener/resource_event_handler.go

```diff
@@ -186,21 +178,21 @@ func (k *listenerImpl) handleAllEvents() {
 		handle(k.context, complianceScanInformer, dispatchers.ForComplianceOperatorScans(), k.outputQueue, &syncingResources, noDependencyWaitGroup, stopSignal, &eventLock)
 	}
 
-	if !startAndWait(stopSignal, noDependencyWaitGroup, sif, resyncingSif, osConfigFactory, osOperatorFactory, crdSharedInformerFactory) {
+	if !startAndWait(stopSignal, noDependencyWaitGroup, sif, sif, osConfigFactory, osOperatorFactory, crdSharedInformerFactory) {
```

Comment:
No need to add the informer `sif` twice here
```suggestion
	if !startAndWait(stopSignal, noDependencyWaitGroup, sif, osConfigFactory, osOperatorFactory, crdSharedInformerFactory) {
```

Reply:
Hazards of auto rename. Nice catch!